.github/workflows/javaTests.yml

```diff
@@ -176,14 +102,14 @@ jobs:
           ${{ runner.os }}-maven-test-
 
     - name: ${{ matrix.tests }}
-      uses: ./.github/action/
-      id: test
-      with:
-        test-to-run: ${{ matrix.tests }}
```

Comment:
Not sure why the jobs are failing, but this might have something to do with it, I'll check when I have some time this week.